### PR TITLE
Relocate to link relationship to HTML spec

### PR DESCRIPTION
**Pull request by marcoscaceres**
Part of #24389

### CODE REVIEW

File: html/links/manifest/link-relationship/link-tree-order-manual.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<title>Test that name member is supported</title>
```
Comment (christianliebel) [inline html/links/manifest/link-relationship/link-tree-order-manual.html:2]:
```suggestion
<title>Test that the first "manifest" `link` is used for installing</title>
```

**Review by christianliebel — CHANGES_REQUESTED** [review]
Rename test title to match the primary testing objective

**Review by christianliebel — APPROVED** [review]

**Comment by stephenmcgruer** [comment]
@marcoscaceres the checks for this PR will be blocked due to the Taskcluster Checks migration (sorry!). You can either rebase/merge-from-master the PR to bring in the new Taskcluster config, or just let me know when you want it merged and I can admin-merge it for you :)

**Comment by marcoscaceres** [comment]
Thanks again @stephenmcgruer! rebased :)

**Pull request merged**